Core/src/main/java/com/theprogrammingturkey/comz/game/features/Door.java

```diff
@@ -185,6 +185,11 @@ public boolean arePointsFinal()
 		return arePointsFinal;
 	}
 
+	public boolean hasDoorBlocks()
+	{
+		return !blocks.isEmpty() && blocks.size() >= 2;
```

Comment:
Pretty sure you only need `blocks.size() >= 2` as that also infers the array isn't empty

Reply:
Done